app/views/newPaymentFlow.scala.html

```diff
@@ -0,0 +1,114 @@
+<header role="banner">
+  <a href="#"><img src="#" alt="The Guardian"></a>
+</header>
+
+<h1>Help us deliver the independent journalism the world needs</h1>
+<p>Make a monthly commitment to support The Guardian long term or a one-off contribution as and when you feel like it. Choose the option that suits you best.</p>
+
+<form action="#">
+  <fieldset>
+    <legend>Recurrence</legend>
+    <ul>
+      <li>
+        <input id="contributionType-monthly" type="radio" name="contributionType" value="montly">
+        <label for="contributionType-monthly">Monthly</label>
+      </li>
+      <li>
+        <input id="contributionType-oneoff" type="radio" name="contributionType" value="oneoff">
+        <label for="contributionType-oneoff">One off</label>
+      </li>
+    </ul>
+  </fieldset>
+
+  <fieldset>
+    <legend>Amount</legend>
+    <ul>
+      <li>
+        <input id="contributionAmount-25" type="radio" name="contributionAmount" value="25">
+        <label for="contributionAmount-25" aria-label="25 US Dollars">$25</label>
+      </li>
+      <li>
+        <input id="contributionAmount-50" type="radio" name="contributionAmount" value="50">
+        <label for="contributionAmount-50" aria-label="50 US Dollars">$50</label>
+      </li>
+      <li>
+        <input id="contributionAmount-100" type="radio" name="contributionAmount" value="100">
+        <label for="contributionAmount-100" aria-label="100 US Dollars">$100</label>
+      </li>
+      <li>
+        <input id="contributionAmount-250" type="radio" name="contributionAmount" value="250">
+        <label for="contributionAmount-250" aria-label="250 US Dollars">$250</label>
+      </li>
+      <li>
+        <input id="contributionAmount-other" type="radio" name="contributionAmount" value="other">
+        <label for="contributionAmount-other">Other</label>
+      </li>
+    </ul>
+  
+    <div>
+      <label for="contributionOther">Other Amount</label>
+      <input id="contributionOther" type="text" pattern="[1-9]\d*(.\d{2})?">
+    </div>
+  </fieldset>
+
+  <div>
+    <label for="contributionFName">First Name</label>
+    <input id="contributionFName" type="text" required>
+  </div>
+
+  <div>
+    <label for="contributionLName">Last Name</label>
+    <input id="contributionLName" type="text" required>
+  </div>
+
+  <div>
+    <label for="contributionEmail">Email address</label>
+    <input id="contributionEmail" type="email" required>
+  </div>
+
+  <div>
+    <label for="contributionState">Select your state</label>
```

Comment:
Weird that this is the only label using a verb (cc @ionamckendrick )

Reply:
I agree, actually. This was a hangover from what we currently have. I'd suggest changing it to "State" moving forward. Good spot!